tests/baselines/reference/declarationEmit_UnknownImport.errors.txt

```diff
@@ -1,14 +1,17 @@
 tests/cases/compiler/declarationEmit_UnknownImport.ts(2,1): error TS2304: Cannot find name 'SomeNonExistingName'.
+tests/cases/compiler/declarationEmit_UnknownImport.ts(2,14): error TS2304: Cannot find name 'SomeNonExistingName'.
 tests/cases/compiler/declarationEmit_UnknownImport.ts(2,14): error TS2503: Cannot find namespace 'SomeNonExistingName'.
 tests/cases/compiler/declarationEmit_UnknownImport.ts(2,14): error TS4000: Import declaration 'Foo' is using private name 'SomeNonExistingName'.
 
 
-==== tests/cases/compiler/declarationEmit_UnknownImport.ts (3 errors) ====
+==== tests/cases/compiler/declarationEmit_UnknownImport.ts (4 errors) ====
     
     import Foo = SomeNonExistingName
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 !!! error TS2304: Cannot find name 'SomeNonExistingName'.
                  ~~~~~~~~~~~~~~~~~~~
+!!! error TS2304: Cannot find name 'SomeNonExistingName'.

```

Comment:
extra `cannot find name 'name'` errors look annoying and not very useful, can we do something to suppress them?